extra-dev/packages/coq-mathcomp-multinomials/coq-mathcomp-multinomials.dev/opam

```diff
@@ -1,14 +1,16 @@
 opam-version: "2.0"
-name: "coq-mathcomp-multinomials"
 maintainer: "pierre-yves@strub.nu"
 homepage: "https://github.com/math-comp/multinomials"
 bug-reports: "https://github.com/math-comp/multinomials/issues"
 dev-repo: "git+https://github.com/math-comp/multinomials.git"
 license: "CeCILL-B"
 authors: ["Pierre-Yves Strub"]
-build: [ "dune" "build" "-p" name "-j" jobs ]
+build: [
+  [ "bash" "./configure" ]
```

Comment:
```suggestion
  [ "./configure" ]
```
Does this really have to be bash? Any shell should do for the invocation, right?

Reply:
I think your suggestion makes sense but I just copy-pasted it from math-comp/multinomials#45. cc: @erikmd

Reply:
Let's remove it and see how it goes anyway.